task.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir task && mkdir task/dir1 task/dir2 task/dir3 && mkdir task/dir3/dir4
```

Comment:
Можно соединить в одну команду, а при использовании флага `-p` можно не создавать промежуточные папки
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```